Makefile

```diff
@@ -105,32 +107,39 @@ isort-ci:
 
 pylint: compose-build
 	@echo "Running pylint checks..."
-	@docker-compose run $(IMAGE_NAME) \
+	@docker-compose run \
+	  	-e ANALYTICS_OPT_OUT \
```

Comment:
Nit: a few extra spaces here
```suggestion
	-e ANALYTICS_OPT_OUT \
```

Reply:
Confused.. still not seeing the extra spaces
<img width="992" alt="Screen Shot 2022-06-07 at 2 15 57 PM" src="https://user-images.githubusercontent.com/7697292/172453975-37a5c307-f2c8-4fdb-a80b-416870993097.png">

Reply:
confused... still not seeing the extra spaces on my end:
<img width="981" alt="Screen Shot 2022-06-07 at 2 17 28 PM" src="https://user-images.githubusercontent.com/7697292/172454276-cb0076a2-09a4-41ec-86bd-ae57e3986abf.png">